cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaReconcilerStatusTest.java

```diff
@@ -181,6 +184,87 @@ public void testKafkaReconcilerStatus(VertxTestContext context) {
         }));
     }
 
+    @Test
+    public void testKafkaReconcilerStatusUpdateVersion(VertxTestContext context) {
```

Comment:
Should we have a test here that failed reconciliation does not change the status?